tests/config.yaml

```diff
@@ -11,6 +11,7 @@ testdependecies:
     rpms:
         - python2-expect
         - telnet
+default_module: docker
```

Comment:
I see, I did not realize it. I would prefer to keep both, what do you think?

Reply:
This needs to be documented. default_module directive is important.

Reply:
it is important, sure and it is documented well on various places:
http://meta-test-family.readthedocs.io/en/latest/user_guide/how_to_write_conf_file.html
and in 
http://meta-test-family.readthedocs.io/en/latest/user_guide/environment_variables.html
and also in commented config:
https://github.com/fedora-modularity/meta-test-family/blob/master/docs/example-config.yaml

it is enough, or you would like to document it also on some other place?